content/02.author-sections.md

```diff
@@ -589,6 +587,17 @@ This is an improved version of my main figure in this interesting study [@doi:10
 
 
 <!-- Rashika Ramola -->
+### Rashika Ramola
+Hi I am Rashika Ramola. I am a PhD student at Northeastern University. This is my first PSB. I like computational biology and I am excited to be here.
```

Comment:
Please add a newline character after every sentence, which makes it easier to provide feedback and track changes.  For instance:
```
Hi I am Rashika Ramola.
I am a PhD student at Northeastern University.
This is my first PSB.
I like computational biology and I am excited to be here.
```